python/huggingfaceserver/huggingfaceserver/model.py

```diff
@@ -0,0 +1,419 @@
+# Copyright 2024 The KServe Authors.
```

Comment:
I think the merge conflict is not resolved correctly

Reply:
this file should no longer exist

Reply:
Ooopsie...